blueocean-dashboard/src/main/js/PipelineRoutes.jsx

```diff
@@ -114,7 +114,7 @@ function onLeaveCheckBackground() {
 }
 
 export default (
-    <Route path="/" component={Dashboard} onChange={persistBackgroundOnNavigationChange}>
```

Comment:
No, since in blueocean-web there's actually an `IndexRedirect` that pushes to `/pipelines`. I wonder if at some point we ought to make the default route an extension point so there isn't a hard-coded relationship between web and dashboard.